layers/meta-resin-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend

```diff
@@ -14,6 +14,7 @@ PRODUCTION_CMDLINE = "dwc_otg.lpm_enable=0 console=null root=LABEL=resin-rootA r
 CMDLINE = "${@bb.utils.contains('DISTRO_FEATURES','development-image',"${DEBUG_CMDLINE}","${PRODUCTION_CMDLINE}",d)}"
 CMDLINE_DEBUG = ""
 
+
```

Comment:
Hi. This PR is still wrong.
Please see my suggestions from the other thread

Reply:
Florin, 
I am sorry, maybe too newbee on GitHub, but I can't decode what you ask me to do... It seems to me I removed the HDD specific arguments and ONLY changed the mount point, no? 
Henry